app/views/guests/show.html.erb

```diff
@@ -0,0 +1,10 @@
+
+<h1>Guest: <%= @guest.name %></h1>
+<h3> Nights: <%= @guest.nights %> </h3>
+<h2>Rooms Stayed In:</h2>
+<% @rooms.each do |room| %>
```

Comment:
And here on line 5, we'd just do @guest.rooms.each do.... since you set up the AR associations so we have access to the rooms.